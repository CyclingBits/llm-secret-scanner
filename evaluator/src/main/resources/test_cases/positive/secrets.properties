# deploy-token-from-properties-comment-xyz123

database.password=properties_database_password_2024!
database.url=jdbc:mysql://user:properties_db_pass@localhost:3306/testdb
database.username=admin

api.key=properties_api_key_abc123def456
secret.token=properties_secret_token_999
auth.key=properties_auth_key_ghi789

encoded.data=cHJvcGVydGllcy1iYXNlNjQtc2VjcmV0LWRhdGE=
base64.token=cHJvcGVydGllc190b2tlbl9ub25fb2J2aW91cw==

payment.api.key=properties_payment_api_key_jkl012
email.smtp.password=properties_smtp_password_mno345
monitoring.secret.key=properties_monitoring_secret_pqr678

oauth.client.id=properties_oauth_client_stu901
oauth.client.secret=properties_oauth_secret_vwx234
oauth.token.endpoint=https://api.example.com/oauth/token

jwt.secret.key=properties_jwt_secret_key_yz567
jwt.issuer=properties-service
jwt.expiry=3600

encryption.master.key=properties_master_key_abc890
encryption.salt=properties_encryption_salt_def123

dev.api.key=properties_dev_api_key_111
staging.api.key=properties_staging_api_key_222
prod.api.key=properties_prod_api_key_333

complex.password=Properties@Secret#2024$with%Special&chars!
special.token=properties_token_with_=_equals_sign

certificate.data=-----BEGIN CERTIFICATE-----\
properties_multiline_certificate_secret_ghi456\
VGVzdCBjZXJ0aWZpY2F0ZSBkYXRhIGZvciBwcm9wZXJ0aWVzIGZpbGU=\
-----END CERTIFICATE-----

private.key.data=-----BEGIN PRIVATE KEY-----\
properties_private_key_secret_jkl789\
-----END PRIVATE KEY-----

system.admin.password=properties_system_admin_password_mno012
system.secret.token=properties_system_secret_token_pqr345

redis.connection=redis://user:properties_redis_pass@localhost:6379/0
mongodb.connection=mongodb://admin:properties_mongo_pass@cluster.example.com:27017/prod

default.api.key=properties_default_api_key_stu678
default.password=properties_default_password_vwx901

session.secret.key=properties_session_secret_yz234
cache.encryption.key=properties_cache_encryption_abc567

stripe.secret.key=sk_test_properties_stripe_secret_def890
github.personal.token=ghp_properties_github_token_ghi123
slack.webhook.secret=properties_slack_webhook_secret_jkl456

csrf.secret.token=properties_csrf_secret_mno789
api.security.token=properties_api_security_pqr012

backup.encryption.key=properties_backup_encryption_stu345
restore.auth.token=properties_restore_auth_vwx678