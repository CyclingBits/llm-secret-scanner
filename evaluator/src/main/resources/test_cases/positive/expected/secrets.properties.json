[
  {
    "filePath": "secrets.properties",
    "issueNumber": 1,
    "lineNumber": 1,
    "issueType": "Token",
    "secretValue": "deploy-token-from-properties-comment-xyz123"
  },
  {
    "filePath": "secrets.properties",
    "issueNumber": 2,
    "lineNumber": 3,
    "issueType": "Password",
    "secretValue": "properties_database_password_2024!"
  },
  {
    "filePath": "secrets.properties",
    "issueNumber": 3,
    "lineNumber": 4,
    "issueType": "URL Credential",
    "secretValue": "properties_db_pass"
  },
  {
    "filePath": "secrets.properties",
    "issueNumber": 4,
    "lineNumber": 7,
    "issueType": "API Key",
    "secretValue": "properties_api_key_abc123def456"
  },
  {
    "filePath": "secrets.properties",
    "issueNumber": 5,
    "lineNumber": 8,
    "issueType": "Token",
    "secretValue": "properties_secret_token_999"
  },
  {
    "filePath": "secrets.properties",
    "issueNumber": 6,
    "lineNumber": 9,
    "issueType": "Key",
    "secretValue": "properties_auth_key_ghi789"
  },
  {
    "filePath": "secrets.properties",
    "issueNumber": 7,
    "lineNumber": 11,
    "issueType": "Encoded Secret",
    "secretValue": "cHJvcGVydGllcy1iYXNlNjQtc2VjcmV0LWRhdGE="
  },
  {
    "filePath": "secrets.properties",
    "issueNumber": 8,
    "lineNumber": 12,
    "issueType": "Encoded Secret",
    "secretValue": "cHJvcGVydGllc190b2tlbl9ub25fb2J2aW91cw=="
  },
  {
    "filePath": "secrets.properties",
    "issueNumber": 9,
    "lineNumber": 14,
    "issueType": "API Key",
    "secretValue": "properties_payment_api_key_jkl012"
  },
  {
    "filePath": "secrets.properties",
    "issueNumber": 10,
    "lineNumber": 15,
    "issueType": "Password",
    "secretValue": "properties_smtp_password_mno345"
  },
  {
    "filePath": "secrets.properties",
    "issueNumber": 11,
    "lineNumber": 16,
    "issueType": "Key",
    "secretValue": "properties_monitoring_secret_pqr678"
  },
  {
    "filePath": "secrets.properties",
    "issueNumber": 12,
    "lineNumber": 19,
    "issueType": "Secret",
    "secretValue": "properties_oauth_secret_vwx234"
  },
  {
    "filePath": "secrets.properties",
    "issueNumber": 13,
    "lineNumber": 22,
    "issueType": "JWT Secret",
    "secretValue": "properties_jwt_secret_key_yz567"
  },
  {
    "filePath": "secrets.properties",
    "issueNumber": 14,
    "lineNumber": 26,
    "issueType": "Key",
    "secretValue": "properties_master_key_abc890"
  },
  {
    "filePath": "secrets.properties",
    "issueNumber": 15,
    "lineNumber": 27,
    "issueType": "Key",
    "secretValue": "properties_encryption_salt_def123"
  },
  {
    "filePath": "secrets.properties",
    "issueNumber": 16,
    "lineNumber": 29,
    "issueType": "API Key",
    "secretValue": "properties_dev_api_key_111"
  },
  {
    "filePath": "secrets.properties",
    "issueNumber": 17,
    "lineNumber": 30,
    "issueType": "API Key",
    "secretValue": "properties_staging_api_key_222"
  },
  {
    "filePath": "secrets.properties",
    "issueNumber": 18,
    "lineNumber": 31,
    "issueType": "API Key",
    "secretValue": "properties_prod_api_key_333"
  },
  {
    "filePath": "secrets.properties",
    "issueNumber": 19,
    "lineNumber": 33,
    "issueType": "Password",
    "secretValue": "Properties@Secret#2024$with%Special&chars!"
  },
  {
    "filePath": "secrets.properties",
    "issueNumber": 20,
    "lineNumber": 34,
    "issueType": "Token",
    "secretValue": "properties_token_with_=_equals_sign"
  },
  {
    "filePath": "secrets.properties",
    "issueNumber": 21,
    "lineNumber": 36,
    "issueType": "Certificate",
    "secretValue": "-----BEGIN CERTIFICATE-----\\"
  },
  {
    "filePath": "secrets.properties",
    "issueNumber": 22,
    "lineNumber": 41,
    "issueType": "Private Key",
    "secretValue": "-----BEGIN PRIVATE KEY-----\\"
  },
  {
    "filePath": "secrets.properties",
    "issueNumber": 23,
    "lineNumber": 45,
    "issueType": "Password",
    "secretValue": "properties_system_admin_password_mno012"
  },
  {
    "filePath": "secrets.properties",
    "issueNumber": 24,
    "lineNumber": 46,
    "issueType": "Token",
    "secretValue": "properties_system_secret_token_pqr345"
  },
  {
    "filePath": "secrets.properties",
    "issueNumber": 25,
    "lineNumber": 48,
    "issueType": "URL Credential",
    "secretValue": "properties_redis_pass"
  },
  {
    "filePath": "secrets.properties",
    "issueNumber": 26,
    "lineNumber": 49,
    "issueType": "URL Credential",
    "secretValue": "properties_mongo_pass"
  },
  {
    "filePath": "secrets.properties",
    "issueNumber": 27,
    "lineNumber": 51,
    "issueType": "API Key",
    "secretValue": "properties_default_api_key_stu678"
  },
  {
    "filePath": "secrets.properties",
    "issueNumber": 28,
    "lineNumber": 52,
    "issueType": "Password",
    "secretValue": "properties_default_password_vwx901"
  },
  {
    "filePath": "secrets.properties",
    "issueNumber": 29,
    "lineNumber": 54,
    "issueType": "Key",
    "secretValue": "properties_session_secret_yz234"
  },
  {
    "filePath": "secrets.properties",
    "issueNumber": 30,
    "lineNumber": 55,
    "issueType": "Key",
    "secretValue": "properties_cache_encryption_abc567"
  },
  {
    "filePath": "secrets.properties",
    "issueNumber": 31,
    "lineNumber": 57,
    "issueType": "Key",
    "secretValue": "sk_test_properties_stripe_secret_def890"
  },
  {
    "filePath": "secrets.properties",
    "issueNumber": 32,
    "lineNumber": 58,
    "issueType": "Token",
    "secretValue": "ghp_properties_github_token_ghi123"
  },
  {
    "filePath": "secrets.properties",
    "issueNumber": 33,
    "lineNumber": 59,
    "issueType": "Secret",
    "secretValue": "properties_slack_webhook_secret_jkl456"
  },
  {
    "filePath": "secrets.properties",
    "issueNumber": 34,
    "lineNumber": 61,
    "issueType": "Token",
    "secretValue": "properties_csrf_secret_mno789"
  },
  {
    "filePath": "secrets.properties",
    "issueNumber": 35,
    "lineNumber": 62,
    "issueType": "Token",
    "secretValue": "properties_api_security_pqr012"
  },
  {
    "filePath": "secrets.properties",
    "issueNumber": 36,
    "lineNumber": 64,
    "issueType": "Key",
    "secretValue": "properties_backup_encryption_stu345"
  },
  {
    "filePath": "secrets.properties",
    "issueNumber": 37,
    "lineNumber": 65,
    "issueType": "Token",
    "secretValue": "properties_restore_auth_vwx678"
  }
]
